### PR TITLE
[Components with more than 1 service] Refactor Saml components from saml modules

### PR DESCRIPTION
**Pull request by GabrielHenriquedev**

### CODE REVIEW

File: modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/BaseSamlTestCase.java [hunk 1]
```diff
@@ -629,6 +633,9 @@ private void _setupMetadata() throws Exception {
 
 		credentialResolver = new KeyStoreCredentialResolver();
 
+		localEntityManager =
```
Comment (rafaprax) [inline modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/BaseSamlTestCase.java:636]: @GabrielHenriquedev move this variable initialization close to the usage. Thx

File: modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/BaseSamlTestCase.java [hunk 2]
```diff
@@ -618,6 +620,8 @@ private void _setupMetadata() throws Exception {
 
 		fileSystemKeyStoreManagerImpl = new FileSystemKeyStoreManagerImpl();
 
+		samlHttpRequestUtilImpl = new SamlHttpRequestUtilImpl();
+
```
Comment (rafaprax) [inline modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/BaseSamlTestCase.java:624]: Hey @GabrielHenriquedev, move this variable to the `MetadataManagerImplTest.java` class instead. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by GabrielHenriquedev** [comment]
ci:test:sf

**Comment by GabrielHenriquedev** [comment]
ci:test:relevant

**Comment by GabrielHenriquedev** [comment]
ci:test:sf

**Comment by GabrielHenriquedev** [comment]
ci:test:relevant

**Comment by GabrielHenriquedev** [comment]
ci:test:saml

**Comment by GabrielHenriquedev** [comment]
ci:test:security

**Pull request closed without merging**